### DISCUSSION TITLE
Personal dashboard - entries for new releases of repos I don't follow

### DISCUSSION THREAD

**Discussion by darkred** [community/community#4706] [Other Feature Feedback, Questions, & Ideas]
Today I noticed in my personal dashboard *(the first page you'll see when signing in on GitHub, i.e. https://github.com/)* that there are entries for new releases of repos I don't follow (neither themselves nor their author) - I have only starred the repos.

It's these repos: 
https://github.com/gajus/eslint-plugin-jsdoc
https://github.com/vercel/ncc

Screenshots:
<details>
<summary>the dashboard entries:</summary>

![2021-07-18_161407](https://user-images.githubusercontent.com/723651/126068549-406e7328-1020-4a1f-b39d-9a930b897ee1.jpg)

![2021-07-18_181748](https://user-images.githubusercontent.com/723651/126072563-82c83223-7dcb-47bd-982e-6b84103b0d7d.jpg)
</details>

<details>
<summary>the repo pages (the notifications are set to default, i.e. <code>Watch</code>) :</summary>

![2021-07-18_181438](https://user-images.githubusercontent.com/723651/126072481-7115bc8c-dcf7-4610-8904-06cc1a9b8d76.jpg)

![2021-07-18_181629](https://user-images.githubusercontent.com/723651/126072529-a1b606ea-0b1d-4f50-b8c9-48b9c30b36d6.jpg)
</details>

<details>
<summary>the repo authors pages (I don't follow neither) :</summary>

![2021-07-18_161424](https://user-images.githubusercontent.com/723651/126072382-6ee67723-4ee0-427f-b2ee-38e423764e8d.jpg)

![2021-07-18_163005](https://user-images.githubusercontent.com/723651/126069009-0ca07943-fee7-487c-aefe-baecbaf1e62a.jpg)
</details>

According to [the documentation](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-user-account-settings/about-your-personal-dashboard#staying-updated-with-activity-from-the-community) these entries are irrelevant.
Could it be happening simply because these repos match my starred **topics** ? ([ESLint](https://github.com/topics/eslint) and [nodejs](https://github.com/topics/nodejs) respectively)

&nbsp;
**Why is this happening?**
Have you changed it so that, just by starring a repo, you start getting updates for it in the dashboard ?

**Comment by athos** [comment]
Same here. I don't want repos I have only starred to show up in my dashboard. If I really want to track their update, I would start watching them, rather than star them.
